overlord/snapstate/handlers_mount_test.go

```diff
@@ -195,6 +195,63 @@ func (s *mountSnapSuite) TestDoMountSnapError(c *C) {
 	})
 }
 
+func (s *mountSnapSuite) TestDoMountSnapErrorSetupSnap(c *C) {
+	v1 := "name: borken\nversion: 1.0\n"
+	testSnap := snaptest.MakeTestSnapWithFiles(c, v1, nil)
+
+	s.state.Lock()
+	defer s.state.Unlock()
+	si1 := &snap.SideInfo{
+		RealName: "borken-in-setup",
+		Revision: snap.R(1),
+	}
+	si2 := &snap.SideInfo{
```

Comment:
Having two revisions here seems unnecessary. Unless I'm missing something I'd just remove one to make the test easier to follow _or_ add a comment explaining how we use those revisions.